### DISCUSSION TITLE
Consider multi-paned headerbar design

### DISCUSSION THREAD

**Discussion by danirabbit** [elementary/mail#462] [Ideas]
![Screenshot from 2020-02-21 09 44 29@2x](https://user-images.githubusercontent.com/7277719/75057930-ea3a4d00-548e-11ea-9167-8429f3e1f12f.png)

**Comment by danirabbit** [comment]
There's some prior art on Dribbble that looks pretty nice: 

![](https://cdn.dribbble.com/users/563897/screenshots/7897478/media/64e9abd78be1135581bd65e65f11e8bc.png)

![](https://cdn.dribbble.com/users/348016/screenshots/3907900/d2.png)

**Comment by cassidyjames** [comment]
Hmmm… at a glance (or at least my initial reaction is that) I'm not a fan; it loses the anchoring effect of the header bar and makes it less clear where the window can be dragged. I think the style can work great for smaller two-pane apps with fewer actions, but in Mail I don't know that it would be better than a standard header bar.

**Reply by Suzie97** [reply]
Handy allows windows to be dragged from anywhere, so how about integrating that to all elementary apps. I think it makes sense to be able to drag a window from anywhere. Especially now that a lot of users use mobile devices a lot. It might feel more natural in a touchscreen.
